src/python/pants/backend/python/lint/python_fmt.py

```diff
@@ -19,7 +20,7 @@ class PythonFmtTargets(LanguageFmtTargets):
 
 
 @union
-class PythonFmtRequest(FmtRequest):
```

Comment:
Should `FmtRequest` be deleted in `fmt.py`?

Reply:
Yup. :coke: